minidump-stackwalk/http_symbol_supplier.cc

```diff
@@ -236,6 +236,20 @@ namespace {
   string JoinURL(CURL* curl, const string& url, const string& sub) {
     return url + "/" + URLEncode(curl, sub);
   }
+
+  string GetCodeIDAndFileQueryString(CURL* curl, const CodeModule* module) {
+      string code_file = PathnameStripper::File(module->code_file());
+      if (code_file.empty()) {
+        return "";
+      }
+
+      string code_id = PathnameStripper::File(module->code_identifier());
+      if (code_id.empty()) {
+        return "";
+      }
+
+      return "?code_file=" + URLEncode(curl, code_file) + "&code_id=" + URLEncode(curl, code_id);
```

Comment:
Perhaps a useless nit, but should we perhaps omit the `?` there?
Technically a "query string" is the stuff *after* the `?` in a URL. 

Then, we could do something like this:

```c++
// (in HTTPSymbolSupplier::FetchSymbolFile)
...

// pseudo code...
if (!query.empty()) {
    found = url_with_query.find("?");
    if (found > 1) {
         url_with_query += "&";
    } else {
        url_with_query += "?";
    }
    url_with_query += query;
}
```
That would make it possible for us to do something like:
```
./bin/stackwalker --symbols-url=https://symbols.mozilla.org/?use_private=true&other=config
```